config/api.yml

```diff
@@ -1386,6 +1388,8 @@
       :post:
       - :name: edit
         :identifier: catalogitem_edit
+      - :name: create
+        :identifier: catalogitem_new
```

Comment:
why **create** action at the resource level ?  probably only  needed on collection.